src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -122,6 +124,29 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
             return BadRequest(response);
         return CreatedAtRoute("GetArtistById", new { id = response.Result?.Id }, response);
     }
+    /// </summary>
```

Comment:
This should be summary opening tag `<summary>`

Reply:
Resolved